app/views/members/_search.html.erb

```diff
@@ -0,0 +1,4 @@
+<%= form_tag(members_path, method: :get) do %>
+  <%= text_field_tag :query, params[:query] %>
```

Comment:
Yes!  The scope was the key.  I couldn't understand how simply having a search_query method used string parameters.  I also would have rather used:
```<%= form_with scope: :member, url: members_path, method: :get, local: true do |form| %>
  <%= form.text_field :query, sanitize(params["query"]) %>
      <%= form.submit %>
  <% end %>
```
but I couldn't figure out how to sanitize the query for display in the search box.    Thanks for the references, @phlipper